modules/core/kivent_core/managers/animation_manager.pyx

```diff
@@ -0,0 +1,33 @@
+from kivent_core.managers.resource_managers cimport ModelManager
+from kivent_core.managers.resource_managers import texture_manager
+from kivent_core.memory_handlers.block cimport MemoryBlock
+from kivent_core.rendering.animation cimport FrameList
+
+
+cdef class AnimationManager:
+
+    def __init__(self, ModelManager model_manager, allocation_size=1024*10):
+        self.model_manager = model_manager
+        self.allocation_size = allocation_size
+        self.memory_block = None
+        self._animations = {}
+
+    def allocate(self, master_buffer, config_dict):
+        cdef MemoryBlock memory_block = MemoryBlock(self.allocation_size, 1, 1)
```

Comment:
This is never assigned to self.memory_block, causing your later call to creating a FrameList to pass in None as the memory_block